web/README.md

```diff
@@ -1,34 +0,0 @@
-# web
```

Comment:
If we're gonna clean this up, can we at least retain the information in the parent README?

Reply:
Done!